app/src/main/java/com/islaharper/dawnofandroid/data/local/dao/AndroidFlavourDao.kt

```diff
@@ -0,0 +1,24 @@
+package com.islaharper.dawnofandroid.data.local.dao
+
+import androidx.paging.PagingSource
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import com.islaharper.dawnofandroid.domain.model.Flavour
+
+@Dao
+interface AndroidFlavourDao {
+
+    @Query("SELECT * FROM flavour_table ORDER BY id ASC")
```

Comment:
```kotlin
flavour_table
```

Can you please see if there is a good convention to use for db table names?

Reply:
not sure if appending `_table` to the table name is useful:

https://www.sqlshack.com/learn-sql-naming-conventions/

how about `flavour` ?

Reply:
Hi, I was wondering about that myself and there was a fair bit of conflicting information on use of singular or plural and whether to use _table or not as even Android Developer codelab used _table: https://developer.android.com/codelabs/android-room-with-a-view-kotlin#4. Happy to rename table to flavour

Reply:
Thank you that link is useful. Happy to go with whichever convention you want to use